test/unit/util/project-saver-hoc.test.jsx

```diff
@@ -423,7 +423,7 @@ describe('projectSaverHOC', () => {
         expect(mockedOnRemixing).toHaveBeenCalledWith(true);
     });
 
-    test('when starting to remix, onRemixing should be called with param true', () => {
+    test('when starting to remix, onRemixing should be called with param false', () => {
```

Comment:
The test above this one has the same name, so the linter caught this copy-paste error